vulnerabilities/tests/test_data_dump.py

```diff
@@ -23,207 +23,180 @@
-
-        for cve_id in self.CVE_IDS:
-            self.assertTrue(Vulnerability.objects.filter(cve_id=cve_id))
-
-    def test_VulnerabilityReference(self):
-        """
-        Check that all vulnerability references from the test data are stored in the database
-        """
-        for ref in ('ASA-201805-22', 'ASA-201805-23', 'ASA-201805-24', 'ASA-201805-25', 'AVG-708'):
-            self.assertEqual(
-                    len(self.CVE_IDS),
-                    VulnerabilityReference.objects.filter(reference_id=ref).count()
+def test_Vulnerability(setDebianData):
+    """
+    Check that all vulnerabilities from the test data are stored in the database
+    """
+    assert 3 == Vulnerability.objects.count()
+
+    assert Vulnerability.objects.filter(cve_id='CVE-2009-1382')
+    assert Vulnerability.objects.filter(cve_id='CVE-2009-2459')
+    assert Vulnerability.objects.filter(cve_id='CVE-2014-8242')
+
+
+def test_VulnerabilityReference(setDebianData):
+    """
+    Check that no vulnerability references were found in the test data
+    """
+    assert 0 == VulnerabilityReference.objects.count()
+
+
+def test_Package(setDebianData):
+    """
+    Check that all packages from the test data are stored in the database
+    """
+    # There are five rows in Package because currently the models allow duplicates
+    # (see issue #28).
+    assert 5 == Package.objects.count()
+
+    assert Package.objects.filter(name='mimetex')
+
+    pkg = Package.objects.get(name='librsync')
+    assert '0.9.7-10' == pkg.version
+    assert 'deb' == pkg.type
+    assert 'debian' == pkg.namespace
+    assert 'distro=jessie' in pkg.qualifiers
+
+
+def test_ImpactedPackage(setDebianData):
+    """
+    Check that all impacted packages from the test data are stored in the database
+    """
+    impacted_pkgs = ImpactedPackage.objects.all()
+
+    assert 1 == impacted_pkgs.count()
+
+    ip = impacted_pkgs[0]
+    assert 'librsync' == ip.package.name
+    assert '0.9.7-10' == ip.package.version
+
+
+def test_ResolvedPackage(setDebianData):
+    """
+    Check that all resolved packages from the test data are stored in the database
+    """
+    resolved_pkgs = ResolvedPackage.objects.all()
+    versions = [rp.package.version for rp in resolved_pkgs]
+
+    assert 4 == resolved_pkgs.count()
+    assert 'mimetex' == resolved_pkgs[0].package.name
+    assert '1.50-1.1' in versions
+    assert '1.74-1' in versions
+
+
+def test_data_dump(setUbuntuData):
+    """
+    Check basic data import
+    """
+    assert Vulnerability.objects.filter(cve_id='CVE-2002-2439')
+    pkgs = Package.objects.filter(name='gcc-4.6')
+    assert pkgs
+
+    pkg = pkgs[0]
+    assert 'deb' == pkg.type
+    assert 'ubuntu' == pkg.namespace
+
+
+CVE_IDS = ('CVE-2018-11362', 'CVE-2018-11361', 'CVE-2018-11360',
+           'CVE-2018-11359', 'CVE-2018-11358', 'CVE-2018-11357',
+           'CVE-2018-11356', 'CVE-2018-11355', 'CVE-2018-11354')
+
+
+def test_Vulnerability(setArchLinuxData):
```

Comment:
This one overwrites the previous `test_Vulnerability` for Debian, right? I would suggest just keeping the `TestCase` based implementation of the tests in this file. But feel free to resolve it differently, as long as no tests are accidentally removed. :)